src/YTableActionHandler.h

```diff
@@ -0,0 +1,47 @@
+/*
+  Copyright (C) 2020 SUSE LLC
+
+  This library is free software; you can redistribute it and/or modify
+  it under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation; either version 2.1 of the
+  License, or (at your option) version 3.0 of the License. This library
+  is distributed in the hope that it will be useful, but WITHOUT ANY
+  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+  FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+  License for more details. You should have received a copy of the GNU
+  Lesser General Public License along with this library; if not, write
+  to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+  Floor, Boston, MA 02110-1301 USA
+*/
+
+#ifndef YTableActionHandler_h
+#define YTableActionHandler_h
+
+#include <functional>
+
+#include "YTable.h"
+#include "YTableItem.h"
+
+#define TablePathDelimiter "|"
+
+class YTableActionHandler
+{
+public:
+
+    YTableActionHandler() {}
+    virtual ~YTableActionHandler() {}
+
+    static std::function<void (YTable*)> get_handler( YTable * widget,
+                                                      const std::string &value,
+                                                      const int &column_id = 0,
```

Comment:
Could you please add a comment for these "magic" default values? What do they mean? It's a bit confusing why `0` and `-1` are used, esp. why here there is `column_id = 0` and in the function below there is `column_id = -1` default.

Reply:
Yeah, it's confusing as `0` is used by default in libyui parent library, and `-1` will mean that we will get default value. But we should be consistent, so I will take care of that.